src/command_modules/azure-cli-network/azure/cli/command_modules/network/_params.py

```diff
@@ -786,6 +786,11 @@ def load_arguments(self, _):
         c.argument('log_format', options_list='--format', help='File type of the flow log.', arg_type=get_enum_type(FlowLogFormatType))
         c.argument('log_version', help='Version (revision) of the flow log.', type=int)
 
+    with self.argument_context('network watcher flow-log', arg_group='Traffic Analytics', min_api='2018-10-01') as c:
+        c.argument('traffic_analytics_interval', arg_type=ignore_type, options_list='--interval', help='Interval in minutes at which to conduct flow analytics.', min_api='2018-12-01')
+        c.argument('traffic_analytics_workspace', options_list='--workspace', help='Name or ID of a Log Analytics workspace.')
+        c.argument('traffic_analytics_enabled', options_list='--traffic-analytics', arg_type=get_three_state_flag(), help='Enable traffic analytics. If supplying `--workspace`, defaults to true.')
```

Comment:
The second statement "if supplying..." sounds a bit strange to me. Maybe something like "Defaults to true, if `if --workspace` is set"